client/app/hosting/ssl/hosting-ssl.service.js

```diff
@@ -0,0 +1,98 @@
+angular
+    .module("services")
+    .service("hostingSSLCertificate", class HostingSSLCertificate {
+        constructor ($rootScope, hostingSSLCertificateType, OvhApiHostingWebSsl) {
+            this.$rootScope = $rootScope;
+
+            this.hostingSSLCertificateType = hostingSSLCertificateType;
+            this.OvhApiHostingWebSsl = OvhApiHostingWebSsl;
+        }
+
+        creatingCertificate (serviceName, certificate, key, chain) {
+            return this.OvhApiHostingWebSsl.Lexi().post({ serviceName }, { certificate, key, chain }).$promise;
```

Comment:
@antleblanc -> /!\ Lexi

Reply:
thx @jleveugle 
I'll put my attention to this part.